client/src/pages/Home/Forms/DonatorFormReqest/DonatorForm.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { useState } from "react";
 import "./DonatorForm.css";
+import { nanoid } from "nanoid";
```

Comment:
I think you're missing the package.json in this commit

Reply:
It must have been merged with the other branch. Because I just took out the functionality.